src/content/en/2021/cdn.md

```diff
@@ -132,27 +135,29 @@ While we observed CDN adoption across different types of content, we will not lo
   )
 }}
 
-Ranking the domains based on their popularity (at the time of this writing sourced from Google CRUX data) in the web and then checking for their CDN usage, the top 1000 websites contribute to the highest usage of CDN. The top websites include domains owned by names like Google, Amazon who contribute to much of the internet traffic we see today and it's no surprise that these names make it to the list of "Top CDN Providers" in the next section. This also backs the fact about the benefits CDN's bring to the table when operating at scale and having the ability to scale further if needed.
+Ranking the domains based on their popularity (at the time of this writing sourced from Google CRUX data) in the web and then checking for their CDN usage, the top 1000 websites contribute to the highest usage of CDN. The top websites include domains owned by names like Google and Amazon who contribute to much of the internet traffic we see today, so it's no surprise that these names make it to the list of "Top CDN Providers" in the [next section](#top-CDN-providers). This also backs the fact about the benefits CDNs bring to the table when operating at scale and having the ability to scale further if needed.
 
 The CDN adoption rate falls below 50% when we look at the top 100,000 websites but the rate of reduction slows down beyond this. For the full data set (which is 6.2 million sites on desktop and 7.5 million on mobile), 27% of these websites use CDN. When you translate that percentage into real number, that's
 
 2.7 million websites using CDN! That is not a small number when you look at it this way.
 
-But the decreasing percentage of CDN adoption in the low popularity website end does make sense considering the benefits of CDN (like caching, TCP connection offload etc.,) increases with increased end users on the web property. Below a certain scale of end user traffic on a web property, the cost-to-benefit math of a CDN may not work in web property owner's favor and they might be better off delivering the web content directly from origin.
+{# TODO: this sentence being broken off seems like a typo. I can merge it back to the previous paragraph but this might also be a good spot to put a big number figure if you want to highlight that number #}
```

Comment:
@Navaneeth-akam

Reply:
yes. its a typo. But your suggestion is good. We can put 2.7 million as a big number and highlight it. That will get some attention

Reply:
I haven't added images before, are you able to commit an update here to add it in?

Reply:
I don't have a big number figure as such. I was thinking more along the lines of increased font size, making them bold and centering it between the paragraph. Does this make sense?

Reply:
Else, we can just make the 2.7 million in bold as a highlight if a figure is not doable

Reply:
there's a reference for it here https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Figures-Guide#call-out-figures
maybe @tunetheweb can help?

Reply:
@Navaneeth-akam if it exists in the [results sheet](https://docs.google.com/spreadsheets/d/1DL7Pn1vbBwYmQZ5JPAjD69oCOvUidbuoNvdrw--j00U/edit) it's easy to make it into a big number figure.

In the cdn_usage_by_site_rank tab I see 2,012,646 mobile "requests" (should this be "pages"?) that use a CDN in the ALL ranking. Would 2M be more accurate or did you get 2.7M from a different query?

Reply:
i calculated 2.7 million looking at the numbers in the results. You will see that for "Top 10 million site", 27% use CDN. Which translates to 2.7 million sites out of 10 million use CDN.

Reply:
Oh ok there are only 7,491,736 pages in the "all" rank, which is why we avoid calling it 10M.

Reply:
Ahh....ok. Gotcha. Then may be we can take out that big number thing. Because we are now quoting top 10 million sites, so trying to take 27% of 7,491,736 will not be equal to 27% of 10 million, Thus it might be confusing for readers how we came up with this number. Thoughts?

Reply:
Why not just put 27% in a big number?

Reply:
That thought did cross my mind after i commented :) We can do that. Works for me. 👍🏼